sdk/src/esplora.ts

```diff
@@ -414,7 +422,18 @@ export class EsploraClient {
 
         const chainBalance = response.chain_stats.funded_txo_sum - response.chain_stats.spent_txo_sum;
         const mempoolBalance = response.mempool_stats.funded_txo_sum - response.mempool_stats.spent_txo_sum;
-        return chainBalance + mempoolBalance;
+
+        console.log({
+            chain: chainBalance,
+            mempool: mempoolBalance,
+            total: chainBalance + mempoolBalance,
+        });
+
```

Comment:
do you want to keep this?

Reply:
I'd convert at least to `debug` level so it's not shown by default in the browser console.